dmoj/checkers/linecount.py

```diff
@@ -6,8 +6,8 @@
 verdict = u"\u2717\u2713"
 
 
-def check(process_output, judge_output, point_value, feedback=True,
-          match=lambda p, j: p.strip() == j.strip(), **kwargs):
+def check(process_output: str, judge_output: str, point_value: float, feedback: Optional[bool] = True,
+          match: Optional[callable] = lambda p, j: p.strip() == j.strip(), **kwargs) -> Union[CheckerResult, bool]:
```

Comment:
From https://github.com/DMOJ/judge/blob/master/dmoj/checkers/linecount.py#L20, `match` can also be a string.

Reply:
Done!